crates/pallet-balanced-currency-swap-bridges-initializer/src/lib.rs

```diff
@@ -96,6 +93,13 @@ pub mod pallet {
         /// The evm-native bridge pot account.
         type EvmNativeBridgePot: Get<Self::EvmAccountId>;
 
+        /// The current bridges initializer version.
+        type InitializerVersion: Get<u16>;
+
+        /// Whether currencies balanced check required on runtime upgrade.
+        #[pallet::constant]
+        type IsBalancedCheckRequiredOnRuntimeUpgrade: Get<bool>;
```

Comment:
This should be more expressive. What happens if the check is not required? Is it skipped? Let's just call it `SkipBalanceVerificationOnRuntimeUpgrade`

Reply:
Renamed.